### DISCUSSION TITLE
Raid, backup, rebuild

### DISCUSSION THREAD

**Discussion by cremonies** [007revad/Synology_HDD_db#386] [Q&A] [answered]
I'm setting this drive for Plex and syno apps and wanted to know if I should use 2 drives for redundancy.

Since I'm doing this for the performance boost of Plex loading thumbnails, and all my movies are on the HDDs, should I still set write_mostly to HDDs?

**Comment by 007revad** [comment]
I prefer 2 drives for redundancy.

write_mostly only works if the SSD drives are 2.5 inch drives. M.2 drives do not have DSM mirrored to them.

**Reply by cremonies** [reply]
That's good to know. 
Is there any warnings that M.2 drives are failing? 
Don't know if I should move Synology photo over and have the possibility of loosing albums.

**Reply by 007revad** [reply, marked as answer]
M.2 drives tend to die suddenly, usually after they reach their TBW limit. 

I probably would not move Synology photos or `@database`. Moving the database to the NVMe drive(s) is risky because if after a DSM update DSM does not mount the NVMe volume DSM will create a new, empty, database on the HDD volume.
